### DISCUSSION TITLE
Windows - allow to embed configuration in the executable

### DISCUSSION THREAD

**Discussion by mterwoord** [rustdesk/rustdesk#4693] [Feature Request]
Hi,

On windows, it's possible to add random data to a .exe file (by concatting) 
Of course this breaks the signature, but this would be a great way to allow embedding the license/configuration info in the file, without having to use an obscure filename.

I'd be happy to work on it myself. Would this be something that would be merged, if a PR would be made?

With kind regard, 
Matthijs ter Woord

**Comment by Vlod-github** [comment]
Embedding data in an executable doesn't look like something good

**Comment by deep-soft** [comment]
If you really want to add info to exe file maybe is better to add an "resource" (similar with FILEVERSION, see Runner.rc) and sign again the executable with your own key.
